### PR TITLE
Add new hypertrophic anatomical entity pattern

### LINKED ISSUE #795: Add new hypertrophic anatomical entity pattern
1. [ ] add new pattern `*.yaml` for review
2. [ ] get feedback
3. [ ] fix or improve pattern based on feedback

* Examples:
    1. [HP:0003712](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0003712)	[Skeletal muscle hypertrophy](http://purl.obolibrary.org/obo/HP_0003712)
    2. [MP:0009461](https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMP_0009461)	[skeletal muscle hypertrophy](http://purl.obolibrary.org/obo/MP_0009461)
    3. [MP:0000053](https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMP_0000053)	[excessive ear growth](http://purl.obolibrary.org/obo/MP_0000053)

### PR DESCRIPTION
**Pull request by rays22**
Add `hypertrophicAnatomicalEntity.yaml` DOSDP pattern template.

If applied, this commit will fix #795.

### CODE REVIEW

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
Do you need my ORCID as a contributor? It is:
https://orcid.org/0000-0001-5208-3432

**Comment by rays22** [comment]
> Do you need my ORCID as a contributor? It is: https://orcid.org/0000-0001-5208-3432

Yes, thank you. :)

**Comment by sbello** [comment]
examples
http://www.informatics.jax.org/vocab/mp_ontology/MP:0005603 neuron hypertrophy

**Comment by sbello** [comment]
change this to tissue hypertrophy as there is no difference for single cells between increased cell size and cell hypertrophy

File: src/patterns/dosdp-dev/hypertrophicAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - anatomical_entity
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "%s hypertrophy"
+    vars:
+      - anatomical_entity
+
+def:
+  text: "An abnormal overgrowth of %s."
```
Comment (dosumis) [inline src/patterns/dosdp-dev/hypertrophicAnatomicalEntity.yaml:44]: Suggested change to indicate mechanism: 
"Increased size of %s due to overgrowth and enlargement of its constituent cells."

File: src/patterns/dosdp-dev/hypertrophicAnatomicalEntity.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+pattern_name: hypertrophicAnatomicalEntity
+
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/hypertrophicAnatomicalEntity.yaml
+
+description: 'Abnormal overgrowth of anatomical entity, e.g. skeletal muscle hypertrophy.'
```
Comment (dosumis) [inline src/patterns/dosdp-dev/hypertrophicAnatomicalEntity.yaml:5]: Turn this into useful guidance for editors applying the pattern.  Suggested:

"Use this pattern for overgrowth of multicellular structures due to overgrowth and enlargement of their cells. When the mechanism is unknown or for enlargement of any other type of structure, including enlargement of single cells, please use abnormallyIncreasedSizeOfAnatomicalEntity."

**Pull request merged**